lucene/core/src/java/org/apache/lucene/store/ByteBuffersIndexInput.java

```diff
@@ -193,5 +193,10 @@ private void ensureOpen() {
     if (in == null) {
       throw new AlreadyClosedException("Already closed.");
     }
-  }  
+  }
+
+  @Override
+  public boolean isMMapped() {
+    return true;
```

Comment:
Hi Simon. Whether this should return true depends on what byte buffers are used? The same applies to ByteBufferIndexInput, actually... I don't think you can generally tell whether the ByteBuffers the input operates on come from a mmap call or from somewhere else (even direct buffers don't have to be a result of mmap).

Reply:
it's rather mmapped or in ram so `true` is fine?

Reply:
Err... I didn't follow this change too closely. I thought the meaning of this is to distinguish between memory-mapped (off-heap) vs. heap-based storage?

Reply:
I will clarify the API.

Reply:
I removed this API went down a different route